.github/workflows/ci.yml

```diff
@@ -18,14 +18,34 @@ jobs:
             java: 8
     steps:
       - uses: actions/checkout@v2
-      - uses: olafurpg/setup-scala@v13
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: ${{ matrix.java }}
+      # We need to set proper Pagefile limits in advance.
```

Comment:
Take from Scala Native CI

Reply:
Had no idea about this, I'm about to setup some Windows SN workflows, so it's cool to know :)